### DISCUSSION TITLE
Antigen needs to use an extra branch tag when installing

### DISCUSSION THREAD

**Discussion by cxz66666** [z-shell/zi#269] [General]
I use antigen v2.2.2 to install z-shell bundles, including `zsh-navigation-tools` and `zsh-cmd-architect`. But I found need to use
```bash
antigen bundle z-shell/zsh-cmd-architect@main
antigen bundle z-shell/zsh-navigation-tools@main
```
Need to specify the default branch because antigen only regards the `master` branch as  default branch.

Please update Readme on those repos, THX!

**Comment by ss-o** [comment]
Hi @cxz66666 :wave: 

Thanks for pointing this out :dart: Could you confirm that the changes are correct?

- https://github.com/z-shell/zsh-navigation-tools/pull/4/commits/84ccded6e810369d9c6b09235df85c5113332862#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R86
- https://github.com/z-shell/zsh-cmd-architect/pull/5/commits/3ed9380987b3d9965cc04b17ec74fece5a6b2a5f#diff-0b5ca119d2be595aa307d34512d9679e49186307ef94201e4b3dfa079aa89938R50

**Reply by cxz66666** [reply]
Wow, that's very nice. Thank you 💯! I don't take all looks of repos in `Z-Shell` organization, there may be some omissions.

**Reply by ss-o** [reply]
Please feel free to report any issues, and propose suggestions/changes to any repository :+1:
